Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -173,7 +178,61 @@ After loading the script, a window that contains two sections, ``SCHEMA`` (top)
 - ``oods_timeout``: Timeout value, in seconds, for the Observatory Operations Data Service (``OODS``).
   Default: ``120``.
 
-An example set of configuration parameters is as follows:
+
+Configuration examples
+----------------------
+
+**Preferred daily script mode to be run**: if no configuration parameters are passed to LOVE and the default parameters are used, the script will take 21 biases, 21 darks of 5 seconds each one, and 21 flats of 5 seconds each one. In each case, the first image will be discarded. New combined calibrations will not be generated, and verification of the images taken will be performed using the existing combined calibrations in the ``LATISS/calib`` collection (i.e., the script will do ``external verification``). In this case, no defects will be made. Following DMTN-222, a gain estimate will be produced from each of the 10 flat pairs taken. **Users should adjust parameters when needed, for example, the exposure times or the number of exposures taken**.
```

Comment:
I think it'd be good to make each example a subsection
```
Daily Default
^^^^^^^^^^^^^^^^^^
```
And same for the ones below